tests/testProjectClassToR1.m

```diff
@@ -0,0 +1,286 @@
+classdef testProjectClassToR1 < matlab.unittest.TestCase
+
+    properties
+        inputStandardProjectClass
+        inputStandardStruct
+        outputStandardStruct
+        defaultProject
+        outputCustomStruct
+        inputCustomProjectClass
+        modelFile
+        fullPath
+    end
+
+    methods(TestClassSetup)
+        function defineVariables(testCase)
+            testCase.inputStandardProjectClass = "originalDSPCBilayerProjectClass.mat";
+            testCase.inputStandardStruct = "originalDSPCBilayerStructInput.mat";
+            testCase.outputStandardStruct = "originalDSPCBilayerStructOutput.mat";
+            testCase.defaultProject = "defaultProject.mat";
+            testCase.outputCustomStruct = "monolayerVolumeModel.mat";
+            testCase.inputCustomProjectClass = "monolayerVolumeModelProjectClass.mat";
+            testCase.modelFile = "monolayerVolumeCustomLayerModel.m";
+        end
+    end
+
+    methods(TestMethodSetup)
+        function setWorkingFolder(testCase)
+            root = getappdata(0,'root');
+            testCase.fullPath = [root filesep 'tests' filesep 'testProjectClassToR1Conversion'];
+            import matlab.unittest.fixtures.CurrentFolderFixture
+            testCase.applyFixture(CurrentFolderFixture(testCase.fullPath))
```

Comment:
Please use WorkingFolderFixture when possible don't write files into the repo folder

Reply:
I initially was using the WorkingFolderFixture bur I could not figure out how to give it access to repo files. When using projectClassTor1 on a custom layer model, It needs to copy a file and I could not figure out how to do that with WorkingFolderFixture.

Reply:
<img width="993" alt="image" src="https://user-images.githubusercontent.com/47083562/230240972-e0266a25-6513-4a65-8103-445c1b38de96.png">

The other solution I can think of is creating a short custom model .m file from within the test. This function returns the expected values. `projectClassToR1.m` will copy this model file into the newDirectory folder. In this case, the test does not need access to the model file in the repo. I have deleted that file.

I can also now delete the *TestMethodTeardown* method.